*
* Visual Studio Code Server
*
* By using the software, you agree to
* the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
* the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
*
Server bound to /tmp/code-a030960c-dd3f-42ac-a89e-b20774288f78
Extension host agent listening on /tmp/code-a030960c-dd3f-42ac-a89e-b20774288f78

[11:25:18] 




[11:25:18] Extension host agent started.
[11:25:18] [<unknown>][5841e4aa][ExtensionHostConnection] New connection established.
[11:25:18] [<unknown>][0b0c50dc][ManagementConnection] New connection established.
[11:25:18] [<unknown>][5841e4aa][ExtensionHostConnection] <4092436> Launched Extension Host Process.
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty 4 on pty host
stack trace: CodeExpectedError: Could not find pty 4 on pty host
    at I.W (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:46:6451)
    at I.updateIcon (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:46:1889)
    at s.<computed> (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:45:2938)
    at Object.call (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:28:4204)
    at vl.s (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:81574)
    at vl.q (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:81097)
    at ps.value (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80499)
    at E.C (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2373)
    at E.fire (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2591)
    at process.C (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:24:30159)
    at process.emit (node:events:518:28)
    at emit (node:internal/child_process:949:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:91:21)
[11:25:22] Error [CodeExpectedError]: Could not find pty 4 on pty host
    at I.W (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:46:6451)
    at I.updateIcon (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:46:1889)
    at s.<computed> (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:45:2938)
    at Object.call (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:28:4204)
    at vl.s (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:81574)
    at vl.q (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:81097)
    at ps.value (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80499)
    at E.C (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2373)
    at E.fire (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2591)
    at process.C (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:24:30159)
    at process.emit (node:events:518:28)
    at emit (node:internal/child_process:949:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:91:21)
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty 4 on pty host
stack trace: CodeExpectedError: Could not find pty 4 on pty host
    at I.W (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:46:6451)
    at I.updateTitle (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:46:1839)
    at s.<computed> (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:45:2938)
    at Object.call (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:28:4204)
    at vl.s (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:81574)
    at vl.q (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:81097)
    at ps.value (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80499)
    at E.C (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2373)
    at E.fire (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2591)
    at process.C (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:24:30159)
    at process.emit (node:events:518:28)
    at emit (node:internal/child_process:949:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:91:21)
[11:25:22] Error [CodeExpectedError]: Could not find pty 4 on pty host
    at I.W (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:46:6451)
    at I.updateTitle (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:46:1839)
    at s.<computed> (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:45:2938)
    at Object.call (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:28:4204)
    at vl.s (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:81574)
    at vl.q (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:81097)
    at ps.value (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80499)
    at E.C (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2373)
    at E.fire (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2591)
    at process.C (file:///home/b/b92314sp/b24mybeget.ru/public_html/.vscode-server/cli/servers/Stable-6f17636121051a53c88d3e605c491d22af2ba755/server/out/vs/platform/terminal/node/ptyHostMain.js:24:30159)
    at process.emit (node:events:518:28)
    at emit (node:internal/child_process:949:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:91:21)
[11:25:45] [File Watcher ('parcel')] Inotify limit reached (ENOSPC) (path: /home/b/b92314sp/b24mybeget.ru/public_html)
[11:27:44] [<unknown>][0b0c50dc][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[11:27:44] [<unknown>][5841e4aa][ExtensionHostConnection] <4092436> Extension Host Process exited with code: 0, signal: null.
Cancelling previous shutdown timeout
[11:27:44] Cancelling previous shutdown timeout
Last EH closed, waiting before shutting down
[11:27:44] Last EH closed, waiting before shutting down
